website/integrations/services/dokuwiki/index.md

```diff
@@ -60,6 +60,9 @@ In authentik, under _Providers_, create an _OAuth2/OpenID Provider_ with these s
 
 Note the _client ID_ and _client secret_, then save the provider. If you need to retrieve these values, you can do so by editing the provider.
 
+You will need to include the _offline access_ scope both in authentik and DokuWiki to allow Dokuwiki to use refersh tokens. Otherwise you will need to
```

Comment:
suggested rewrite: "To prevent users from needing to log in again as soon as the access token expires, include the _offline access_ scope in both authentik and DokuWiki. This scope allows Dokuwiki to use refresh tokens."

Also I think there is a line break between lines 63 and 64...?

Reply:
I've rewritten it as suggested and remove the line break. 👍

Reply:
Hi @pkuehne can you run `make website` and then push again, so that prettier gets run? The build is failing on that check. Or let us know if you have and it is still failing... thanks!

Reply:
Hi @tanberry, I ran it again on my branch, but it succeeded and there is nothing to push.

![image](https://github.com/goauthentik/authentik/assets/4983974/dcd34f19-e0db-4427-9e84-7a6b10ce23a3)